content/hi/docs/reference/glossary/sig.md

```diff
@@ -0,0 +1,19 @@
+---
+title: SIG (special interest group)
```

Comment:
- Consider moving “special interest group” to the `aka` section.
- Consider putting a translation of “special interest group” into the `aka` section as well.